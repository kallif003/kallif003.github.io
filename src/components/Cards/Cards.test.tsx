import * as React from "react"
import { render } from "@testing-library/react"
import {
	ContainerCards,
	Cards,
	WorkTools,
	AreaAboutMe,
	AreaTextAboutMe,
} from "."

describe("Cards Components", () => {
	it("renders Container properly", () => {
		const { container } = render(<ContainerCards>Teste</ContainerCards>)
		expect(container.firstChild).toMatchSnapshot()
	})
	it("renders Cards properly", () => {
		const { container } = render(<Cards>Teste</Cards>)
		expect(container.firstChild).toMatchSnapshot()
	})

	it("renders WorkTools properly", () => {
		const { container } = render(<WorkTools>Teste</WorkTools>)
		expect(container.firstChild).toMatchSnapshot()
	})
	it("renders AreaAboutMe properly", () => {
		const { container } = render(<AreaAboutMe>Teste</AreaAboutMe>)
		expect(container.firstChild).toMatchSnapshot()
	})

	it("renders AreaTextAboutMe properly", () => {
		const { container } = render(<AreaTextAboutMe>Teste</AreaTextAboutMe>)
		expect(container.firstChild).toMatchSnapshot()
	})
})
