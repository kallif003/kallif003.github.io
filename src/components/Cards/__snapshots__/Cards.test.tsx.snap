// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cards Components renders AreaAboutMe properly 1`] = `
<div
  class="h-[453.1620178222656px] w-[967px] left-[64px] top-[216px] rounded-[8px] p-[24px] bg-[#1f1f21] flex flex-row md:flex-col md:h-[993px] md:w-[328px] md:left-[0px] md:top-[0px] md:p-[16px] sm:flex-col sm:h-[993px] sm:w-[328px] sm:left-[0px] sm:top-[0px] sm:p-[16px] mini:w-[260px] mini:flex-col mini:h-[900px]"
>
  Teste
</div>
`;

exports[`Cards Components renders AreaTextAboutMe properly 1`] = `
<div
  class="h-[333px] w-[560px] left-[0px] top-[0px] pl-12 text-white md:pl-0 md:w-auto mini:w-auto mini:pl-0 mini:mt-4 sm:w-auto sm:px-10 sm:mt-4"
>
  Teste
</div>
`;

exports[`Cards Components renders Cards properly 1`] = `
<div
  class="h-auto w-[354px] left-[0px] top-[0.045px] rounded-[8px] py-[10px] px-[24px] bg-[#1f1f21] mx-3 mini:w-auto sm:mb-[10px] md:mb-2 lg:mb-[20px] lg:w-[200px]"
>
  Teste
</div>
`;

exports[`Cards Components renders ContainerCards properly 1`] = `
<div
  class="flex flex-row justify-around h-auto pb-[20px] pt-2 px-2 items-center sm:flex-col md:flex-col"
>
  Teste
</div>
`;

exports[`Cards Components renders WorkTools properly 1`] = `
<div
  class="h-[363px] w-[354px] left-[16.5px] top-[19px] rounded-[8px] border-solid border border-[#3d3d3d] md: mg-[20px]"
>
  Teste
</div>
`;
